packages/diagrams/backend/sirius-components-diagrams-tests/src/main/java/org/eclipse/sirius/components/diagrams/tests/builder/node/NodesBuilder.java

```diff
@@ -45,6 +45,12 @@ public RectangleNodeBuilder<T> rectangleNode(String nodeLabel) {
         return nodeBuilder;
     }
 
+    public RectangleNodeBuilder<T> rectangleNode(String nodeLabel, boolean withHeader) {
+        RectangleNodeBuilder<T> nodeBuilder = new RectangleNodeBuilder<>(this, nodeLabel, this.isBorderNode, withHeader);
```

Comment:
I think this is wrong. In `RectangleNodeBuilder` the 4-args constructor expects the "withHeader" before "isBorderNode": `RectangleNodeBuilder`(NodesBuilder<T> nodesBuilder, String nodeLabel, boolean isLabelAHeader, boolean isBorderNode)`.

The fix is probably to switch the order of the two booleans in the constructor, to keep `isBorderNode` in the same (3rd) position in both constructors.

https://ariya.io/2011/08/hall-of-api-shame-boolean-trap